xlb/operator/collision/kbc.py

```diff
@@ -6,12 +6,13 @@
 from jax import jit
 import warp as wp
 from typing import Any
+from functools import partial
 
 from xlb.velocity_set import VelocitySet, D2Q9, D3Q27
 from xlb.compute_backend import ComputeBackend
 from xlb.operator.collision.collision import Collision
 from xlb.operator import Operator
-from functools import partial
+from xlb.operator.macroscopic import SecondMoment
```

Comment:
Why not calling SecondMoment -> MomentumFlux?

Reply:
Second moment is more general. If the input happens to be f-feq then the second moment is called the momentum flux tensor.

Reply:
When is it not the case in LBM?

Reply:
When one wants to compute the second moment of populations.

Reply:
Not sure if we're doing this in XLB now. I suggest changing the Macroscopic class to "Moments" that takes in the order as a static parameter. It's not consistent now.

Reply:
The issue with that is that density is Zero'th moment and rho * u is the first moment! so if we want to do it that way, the kernels for rho and vel becomes separated which is not good. Also we want u and not rho * u and so calling it first moment (for velocity) is also not accurate. I think this is fine for now.

Reply:
In this case‌، if we're not using SecondMoment for anything but computing momentum flux now change it's naming to make it consistent. Thanks.

Reply:
Even if you use it in another manner, you can simply inherit from this and name it differently. So every parameter will have clear name rather than an abstraction.

Reply:
I made the naming consistent.